src/parse.rs

```diff
@@ -966,11 +989,27 @@ fn process_text<'input>(
     Ok(())
 }
 
-enum BorrowedText<'input, 'temp> {
+pub(crate) enum BorrowedText<'input, 'temp> {
     Input(&'input str),
     Temp(&'temp str),
 }
 
+impl<'input, 'temp> BorrowedText<'input, 'temp> {
```

Comment:
This is part of #80 where it required to delay taking onwership of normalized namespace URI. (The added methods are code blocks which would otherwise be repeated in both `append_text` and `Namespaces::push_ns`.)